### PR TITLE
Add custom message for form's regex validation

### PR DESCRIPTION
**Pull request by sukhwinder33445**

### CODE REVIEW

File: application/forms/BpConfigForm.php [hunk 1]
```diff
@@ -25,9 +25,15 @@ public function setup()
                 ),
                 array(
                     'validator' => 'Regex',
-                    'options' => array(
-                        'pattern' => '/^[a-zA-Z0-9](?:[a-zA-Z0-9 ._-]*)?[a-zA-Z0-9_]$/'
-                    )
+                    'options' => [
+                        'pattern' => '/^[a-zA-Z0-9](?:[a-zA-Z0-9 ._-]*)?[a-zA-Z0-9_]$/',
```
Comment (nilmerg) [inline application/forms/BpConfigForm.php:29]: https://github.com/Icinga/icingaweb2-module-businessprocess/pull/354#discussion_r1272140545

**Review by nilmerg — CHANGES_REQUESTED** [review]

File: application/forms/BpConfigForm.php [hunk 2]
```diff
@@ -25,9 +25,15 @@ public function setup()
                 ),
                 array(
                     'validator' => 'Regex',
-                    'options' => array(
-                        'pattern' => '/^[a-zA-Z0-9](?:[a-zA-Z0-9 ._-]*)?[a-zA-Z0-9_]$/'
-                    )
+                    'options' => [
+                        'pattern' => '/^[a-zA-Z0-9](?:[\w\h.-]*)?[a-zA-Z0-9]$/',
```
Comment (nilmerg) [inline application/forms/BpConfigForm.php:29]: The trailing underscore was previously allowed. Was this a mistake here or is there actually a problem with a trailing underscore?

Comment (sukhwinder33445) [reply application/forms/BpConfigForm.php:29]: I have added the missing validators without underscore. The old config will still work with a trailing underscore, but the user should not be able to create a new one. See below:

https://github.com/Icinga/icingaweb2-module-businessprocess/blob/6f1c95f32e397def69f19a564a46fc0064a46093/application/forms/AddNodeForm.php#L94

**Review by nilmerg — CHANGES_REQUESTED** [review]
As discussed, please restore that trailing underscores are allowed.

**Review by nilmerg — APPROVED** [review]

**Pull request merged**